frontend/src/fixtures/coursesFixtures.js

```diff
@@ -87,6 +87,118 @@ const coursesFixtures = {
       studentStatus: "Illegal status that will never occur",
     },
   ],
+  oneRosterStudentWithEachStatus: [
+    {
+      id: 1,
+      rosterStudentId: 1,
```

Comment:
```suggestion
      id: 1,
      rosterStudentId: 21,
```

I suggest that we make these different so that if/when there are coding errors that erroneously interchange them, that becomes more obvious.